iree/test/e2e/vulkan_specific/BUILD

```diff
@@ -74,9 +74,6 @@ iree_check_single_backend_test_suite(
         "vectorized_conv.mlir",
     ],
     compiler_flags = [
-        "-iree-flow-dispatch-linalg-on-tensors",
-        "-iree-codegen-spirv-experimental-linalg-on-tensors",
-        "-iree-spirv-enable-vectorization",
```

Comment:
This flag probably needs to be kept (I am talking about just `iree-spirv-enable-vectorization`?

Reply:
No, we don't need it today. Vectorization is turned on by default in Linalg on tensors, see https://github.com/google/iree/pull/5280